workflows/cloudify_system_workflows/snapshots/npm.py

```diff
@@ -0,0 +1,49 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import os
+
+from cloudify.workflows import ctx
+from .utils import run as run_shell
+
+NODEJS_DIR = os.path.join('/opt', 'nodejs')
+
+
+class Npm(object):
+
+    def __init__(self):
+        self.npm_bin = os.path.join(NODEJS_DIR, 'bin', 'npm')
+
+    @staticmethod
+    def clear_db():
+        """ Clear the Stage DB """
+        ctx.logger.info('Clearing Stage DB')
+        command = ['/opt/nodejs/bin/npm', 'run', 'db-migrate-clear']
+        run_shell(command, cwd='/opt/cloudify-stage/backend')
```

Comment:
Since we are passing this `cwd` in several places (not only this module) - perhaps it would be better to have a utility method for `node/npm` ?